app/assets/javascripts/import_records/ImportRecordCard.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import ImportRecordTimingSection from './ImportRecordTimingSection';
+import ImportRecordToggleableSection from './ImportRecordToggleableSection';
+
+export default class ImportRecordCard extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showOptions: false,
+      showFileByFile: false,
+      showLog: false,
+    };
+  }
+
+  onToggleState(stateKey) {
+    return function () {
+      this.setState({ [stateKey]: !this.state[stateKey] });
+    }.bind(this);
+  }
+
+  render() {
+    const {id} = this.props;
+    const divStyle = {border: '1px solid #eee', padding: 15, margin: '15px 0'};
+    const cardTitleStyle = {marginBottom: 15, borderBottom: '1px solid #eee', fontSize: 18};
+
+    return (
+      <div style={divStyle}>
+        <div style={cardTitleStyle}>Import #{id}</div>
```

Comment:
maybe `<Card />` and `<SectionHeading />` would remove some of the styling duplication?